### PR TITLE
760. Find Anagram Mappings

### PR DESCRIPTION
**Pull request by ErdemT09**
In addition to the HashMap solution we had in the Java repository, I have also though of some space-heavy array solution. I am not sure about its speed though because of implementation differences in C++.

### CODE REVIEW

File: src/leetcode/curated170/easy/find_anagram_mappings/find_anagram_mappings.cpp [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#include <iostream>
+#include <vector>
+#include <unordered_map>
+
+using namespace std;
+
+class Solution
```
Comment (altayhunoglu) [inline src/leetcode/curated170/easy/find_anagram_mappings/find_anagram_mappings.cpp:7]: Its runtime takes 0-8 ms. In some cases, LeetCode is not good at measuring speed.

image

**Review by altayhunoglu — APPROVED** [review]
Thank you Erdem,
Approved.

**Comment by ErdemT09** [comment]
How does the array solution perform?

**Comment by altayhunoglu** [comment]
> How do the both solutions perform?

The other also takes 0-8 ms. Hard to measure.

**Pull request merged**